src/components/chat_lists/ChatList.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Persons from '../persons/Persons';
+import chatLi from './ChatList.module.css';
```

Comment:
import style in the bottom: common order for imports - libraries - components - hooks/utils - constants - styles